Resources/Private/Fusion/Prototypes/Image.fusion

```diff
@@ -45,6 +45,7 @@ prototype(Sitegeist.Kaleidoscope:Image) < prototype(Neos.Fusion:Component) {
     alt = null
     title = null
     class = null
+    outputDimensionsAttributes = true
```

Comment:
`outputDimensionsAttributes` is quite a mouth full. How about `renderDimensions`